tests/integration/test_charm.py

```diff
@@ -78,6 +78,88 @@ async def test_setup_discourse(
     assert response.status_code == 200
 
 
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_db_migration(model: Model, ops_test: OpsTest, pytestconfig: Config, run_action):
+    """
+    arrange: preload postgres with a mock db
+    act: deploy and integrate with discourse
+    assert: discourse must be active idle
+    """
+    postgres_app = await model.deploy(
+        "postgresql-k8s",
+        channel="14/stable",
+        series="jammy",
+        revision=300,
+        trust=True,
+        config={"profile": "testing"},
+    )
+    async with ops_test.fast_forward():
+        await model.wait_for_idle(apps=[postgres_app.name], status="active")
+    # configure postgres
+    await postgres_app.set_config(
+        {
+            "plugin_hstore_enable": "true",
+            "plugin_pg_trgm_enable": "true",
+        }
+    )
+    await model.wait_for_idle(apps=[postgres_app.name], status="active")
+    db_pass = await run_action(postgres_app.name, "get-password", username="operator")
+    db_pass = db_pass["password"]
+    return_code, _, _ = await ops_test.juju(
+        "scp",
+        "--container",
+        # postgres_app.name,
```

Comment:
Is this line needed?

Reply:
Not needed, I missed it when cleaning the code. Thanks for pointing it out!